less-print/viewdetails.less

```diff
@@ -1,5 +1,90 @@
 .view-details-associations, .view-details-comments, .view-details-photos,
-.scroll-top-btn, .view-details-map, .location-static, .badge, .icons-header,
-.logo {
-  display: none;
+.scroll-top-btn, .view-details-map, .badge, .icons-header {
+  display: none !important;
+}
+
+.view-details-title{
+	display: block;
+	font-size: .75em !important;
+	margin-top: 0px;
+	margin-bottom: 10px;
+}
+
+.description , .view-details-info {
+	display: block !important;
```

Comment:
Appart from the comment above, why did you change to `block`? To me it looks like it only adds some unwanted margin.